preDownloadMultipleSecureFiles.ts

```diff
@@ -0,0 +1,50 @@
+import path = require('path');
+import tl = require('azure-pipelines-task-lib/task');
+import secureFilesCommon = require('securefiles-common/securefiles-common');
+
+const commaDelimiter: string = ',';
+const newLineDelimiter: string = '\n';
+
+const main = async () => {
+  try {
+    tl.setResourcePath(path.join(__dirname, 'task.json'));
+
+    let retryCount: number = parseInt(tl.getInput('retryCount') as string);
+
+    if (isNaN(retryCount) || retryCount < 0) {
+      console.log('yes we have nan');
```

Comment:
Probably worth to make it more descriptive e.g. "Couldn't parse value passed for the retryCount parameter (NaN)." or similar

Reply:
Yes also saw it and before you commented I fixed it :)